[2025-08-19T21:50:06.557+0000] {processor.py:161} INFO - Started process (PID=42) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:50:06.561+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:50:06.564+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:50:06.564+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:50:06.585+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:50:06.584+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:50:06.586+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:50:06.611+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.075 seconds
[2025-08-19T21:50:37.165+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:50:37.168+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:50:37.171+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:50:37.171+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:50:37.181+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:50:37.180+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:50:37.182+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:50:37.201+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.041 seconds
[2025-08-19T21:51:07.422+0000] {processor.py:161} INFO - Started process (PID=153) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:51:07.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:51:07.426+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:51:07.426+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:51:07.435+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:51:07.434+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:51:07.435+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:51:07.466+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.052 seconds
[2025-08-19T21:51:37.923+0000] {processor.py:161} INFO - Started process (PID=209) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:51:37.925+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:51:37.927+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:51:37.927+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:51:37.931+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:51:37.930+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:51:37.931+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:51:37.946+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.027 seconds
[2025-08-19T21:52:08.448+0000] {processor.py:161} INFO - Started process (PID=265) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:52:08.450+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:52:08.455+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:52:08.455+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:52:08.466+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:52:08.465+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:52:08.466+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:52:08.497+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.056 seconds
[2025-08-19T21:52:39.298+0000] {processor.py:161} INFO - Started process (PID=321) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:52:39.301+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:52:39.314+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:52:39.313+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:52:39.326+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:52:39.325+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:52:39.326+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:52:39.360+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.075 seconds
[2025-08-19T21:53:09.959+0000] {processor.py:161} INFO - Started process (PID=377) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:53:09.961+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:53:09.963+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:53:09.963+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:53:09.974+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:53:09.971+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:53:09.975+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:53:10.001+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.046 seconds
[2025-08-19T21:53:40.665+0000] {processor.py:161} INFO - Started process (PID=439) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:53:40.668+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:53:40.672+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:53:40.672+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:53:40.680+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:53:40.680+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:53:40.681+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:53:40.712+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.055 seconds
[2025-08-19T21:54:11.281+0000] {processor.py:161} INFO - Started process (PID=495) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:54:11.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:54:11.289+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:54:11.289+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:54:11.304+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:54:11.304+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:54:11.305+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:54:11.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.055 seconds
[2025-08-19T21:54:41.797+0000] {processor.py:161} INFO - Started process (PID=551) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:54:41.800+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:54:41.804+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:54:41.803+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:54:41.822+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:54:41.821+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:54:41.822+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:54:41.842+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.051 seconds
[2025-08-19T21:55:12.400+0000] {processor.py:161} INFO - Started process (PID=607) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:55:12.403+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:55:12.409+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:55:12.409+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:55:12.426+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:55:12.425+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:55:12.426+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:55:12.448+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.056 seconds
[2025-08-19T21:55:43.012+0000] {processor.py:161} INFO - Started process (PID=663) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:55:43.015+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:55:43.018+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:55:43.018+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:55:43.032+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:55:43.032+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:55:43.033+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:55:43.054+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.046 seconds
[2025-08-19T21:56:13.644+0000] {processor.py:161} INFO - Started process (PID=719) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:56:13.647+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:56:13.649+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:56:13.649+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:56:13.659+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:56:13.659+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:56:13.660+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:56:13.684+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.054 seconds
[2025-08-19T21:56:44.261+0000] {processor.py:161} INFO - Started process (PID=775) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:56:44.264+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:56:44.269+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:56:44.269+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:56:44.280+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:56:44.276+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:56:44.281+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:56:44.305+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.050 seconds
[2025-08-19T21:57:14.893+0000] {processor.py:161} INFO - Started process (PID=831) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:57:14.898+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:57:14.901+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:57:14.901+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:57:14.912+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:57:14.911+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:57:14.912+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:57:14.943+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.060 seconds
[2025-08-19T21:57:45.540+0000] {processor.py:161} INFO - Started process (PID=887) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:57:45.542+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:57:45.548+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:57:45.547+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:57:45.554+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:57:45.553+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:57:45.554+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:57:45.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.050 seconds
[2025-08-19T21:58:16.095+0000] {processor.py:161} INFO - Started process (PID=943) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:58:16.097+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:58:16.101+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:58:16.101+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:58:16.107+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:58:16.107+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:58:16.108+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:58:16.133+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.045 seconds
[2025-08-19T21:58:46.647+0000] {processor.py:161} INFO - Started process (PID=999) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:58:46.649+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:58:46.652+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:58:46.651+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:58:46.669+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:58:46.665+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:58:46.671+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:58:46.689+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.047 seconds
[2025-08-19T21:59:17.205+0000] {processor.py:161} INFO - Started process (PID=1055) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:59:17.206+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:59:17.208+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:59:17.208+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:59:17.222+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:59:17.218+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:59:17.224+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:59:17.236+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.036 seconds
[2025-08-19T21:59:47.739+0000] {processor.py:161} INFO - Started process (PID=1111) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:59:47.741+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T21:59:47.744+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:59:47.744+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:59:47.757+0000] {logging_mixin.py:188} INFO - [2025-08-19T21:59:47.756+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T21:59:47.758+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T21:59:47.775+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.041 seconds
[2025-08-19T22:00:18.268+0000] {processor.py:161} INFO - Started process (PID=1167) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:00:18.270+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:00:18.272+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:00:18.272+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:00:18.282+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:00:18.282+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:00:18.283+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:00:18.300+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.037 seconds
[2025-08-19T22:00:48.779+0000] {processor.py:161} INFO - Started process (PID=1223) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:00:48.781+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:00:48.783+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:00:48.783+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:00:48.787+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:00:48.787+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:00:48.788+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:00:48.817+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.053 seconds
[2025-08-19T22:01:19.491+0000] {processor.py:161} INFO - Started process (PID=1279) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:01:19.493+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:01:19.497+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:01:19.496+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:01:19.505+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:01:19.504+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:01:19.506+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:01:19.536+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.049 seconds
[2025-08-19T22:01:49.689+0000] {processor.py:161} INFO - Started process (PID=1335) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:01:49.694+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:01:49.697+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:01:49.696+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:01:49.706+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:01:49.706+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:01:49.707+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:01:49.722+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.048 seconds
[2025-08-19T22:02:20.590+0000] {processor.py:161} INFO - Started process (PID=1391) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:02:20.592+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:02:20.595+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:02:20.595+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:02:20.601+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:02:20.600+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:02:20.601+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:02:20.620+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.040 seconds
[2025-08-19T22:02:51.516+0000] {processor.py:161} INFO - Started process (PID=1447) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:02:51.521+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:02:51.525+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:02:51.525+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:02:51.539+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:02:51.535+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:02:51.539+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:02:51.568+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.057 seconds
[2025-08-19T22:03:22.426+0000] {processor.py:161} INFO - Started process (PID=1503) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:03:22.428+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:03:22.431+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:03:22.430+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:03:22.446+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:03:22.445+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:03:22.448+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:03:22.486+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.067 seconds
[2025-08-19T22:03:53.267+0000] {processor.py:161} INFO - Started process (PID=1559) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:03:53.269+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:03:53.272+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:03:53.272+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:03:53.279+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:03:53.278+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:03:53.279+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:03:53.297+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.037 seconds
[2025-08-19T22:04:23.979+0000] {processor.py:161} INFO - Started process (PID=1615) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:04:23.980+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:04:23.983+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:04:23.982+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:04:23.989+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:04:23.988+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:04:23.989+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:04:24.002+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.027 seconds
[2025-08-19T22:04:54.374+0000] {processor.py:161} INFO - Started process (PID=1671) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:04:54.376+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:04:54.380+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:04:54.379+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:04:54.391+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:04:54.386+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:04:54.391+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:04:54.405+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.035 seconds
[2025-08-19T22:05:24.856+0000] {processor.py:161} INFO - Started process (PID=1727) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:05:24.858+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:05:24.863+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:05:24.862+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:05:24.879+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:05:24.878+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:05:24.879+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:05:24.901+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.053 seconds
[2025-08-19T22:05:54.979+0000] {processor.py:161} INFO - Started process (PID=1783) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:05:54.982+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:05:54.984+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:05:54.983+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:05:54.991+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:05:54.990+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:05:54.991+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:05:55.005+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.031 seconds
[2025-08-19T22:06:25.405+0000] {processor.py:161} INFO - Started process (PID=1839) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:06:25.408+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:06:25.412+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:06:25.412+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:06:25.421+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:06:25.421+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:06:25.425+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:06:25.445+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.047 seconds
[2025-08-19T22:06:55.890+0000] {processor.py:161} INFO - Started process (PID=1895) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:06:55.892+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:06:55.895+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:06:55.895+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:06:55.904+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:06:55.903+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:06:55.904+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:06:55.933+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.048 seconds
[2025-08-19T22:07:25.993+0000] {processor.py:161} INFO - Started process (PID=1951) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:07:25.994+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:07:25.997+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:07:25.997+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:07:26.004+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:07:26.002+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:07:26.005+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:07:26.031+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.048 seconds
[2025-08-19T22:07:56.636+0000] {processor.py:161} INFO - Started process (PID=2007) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:07:56.641+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:07:56.644+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:07:56.644+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:07:56.653+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:07:56.652+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:07:56.653+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:07:56.679+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.051 seconds
[2025-08-19T22:08:27.509+0000] {processor.py:161} INFO - Started process (PID=2063) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:08:27.511+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:08:27.514+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:08:27.514+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:08:27.521+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:08:27.521+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:08:27.522+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:08:27.540+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.037 seconds
[2025-08-19T22:08:58.159+0000] {processor.py:161} INFO - Started process (PID=2119) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:08:58.161+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:08:58.164+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:08:58.164+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:08:58.172+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:08:58.171+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:08:58.172+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:08:58.186+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.033 seconds
[2025-08-19T22:09:28.672+0000] {processor.py:161} INFO - Started process (PID=2175) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:09:28.675+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:09:28.682+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:09:28.681+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:09:28.700+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:09:28.699+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:09:28.701+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:09:28.749+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.085 seconds
[2025-08-19T22:09:59.398+0000] {processor.py:161} INFO - Started process (PID=2231) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:09:59.401+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:09:59.402+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:09:59.402+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:09:59.412+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:09:59.408+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:09:59.412+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:09:59.433+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.039 seconds
[2025-08-19T22:10:29.988+0000] {processor.py:161} INFO - Started process (PID=2287) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:10:29.991+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:10:29.994+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:10:29.994+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:10:30.002+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:10:30.001+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:10:30.002+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:10:30.028+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.047 seconds
[2025-08-19T22:11:00.746+0000] {processor.py:161} INFO - Started process (PID=2343) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:11:00.750+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:11:00.753+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:11:00.752+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:11:00.766+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:11:00.765+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:11:00.766+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:11:00.784+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.044 seconds
[2025-08-19T22:11:31.280+0000] {processor.py:161} INFO - Started process (PID=2399) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:11:31.281+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:11:31.284+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:11:31.284+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:11:31.301+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:11:31.300+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:11:31.301+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:11:31.318+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.043 seconds
[2025-08-19T22:12:01.712+0000] {processor.py:161} INFO - Started process (PID=2455) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:12:01.715+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:12:01.717+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:12:01.717+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:12:01.726+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:12:01.725+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:12:01.727+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:12:01.752+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.047 seconds
[2025-08-19T22:12:32.270+0000] {processor.py:161} INFO - Started process (PID=2511) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:12:32.272+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:12:32.275+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:12:32.274+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:12:32.303+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:12:32.298+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:12:32.306+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:12:32.332+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.082 seconds
[2025-08-19T22:13:02.805+0000] {processor.py:161} INFO - Started process (PID=2567) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:13:02.810+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:13:02.813+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:13:02.813+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:13:02.822+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:13:02.821+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:13:02.822+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:13:02.856+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.061 seconds
[2025-08-19T22:13:33.427+0000] {processor.py:161} INFO - Started process (PID=2623) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:13:33.430+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:13:33.433+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:13:33.432+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:13:33.439+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:13:33.439+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/nyc_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nyc_pipeline_dag.py", line 13, in <module>
    from fetch_311_to_postgres import main as fetch_311
ModuleNotFoundError: No module named 'fetch_311_to_postgres'
[2025-08-19T22:13:33.440+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:13:33.455+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.034 seconds
[2025-08-19T22:20:09.215+0000] {processor.py:161} INFO - Started process (PID=43) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:20:09.217+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:20:09.218+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:09.218+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:20:09.248+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:20:09.467+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:09.466+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:nyc_data_pipeline
[2025-08-19T22:20:09.483+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:09.483+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:nyc_data_pipeline
[2025-08-19T22:20:09.489+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:09.489+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:nyc_data_pipeline
[2025-08-19T22:20:09.490+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:09.490+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:20:09.505+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:09.505+0000] {dag.py:3058} INFO - Creating ORM DAG for nyc_data_pipeline
[2025-08-19T22:20:09.516+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:09.516+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-18 00:00:00+00:00, run_after=2025-08-19 00:00:00+00:00
[2025-08-19T22:20:09.529+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.317 seconds
[2025-08-19T22:20:39.566+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:20:39.570+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:20:39.576+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:39.575+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:20:39.596+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:20:39.616+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:39.616+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:20:39.629+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:20:39.629+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-18 00:00:00+00:00, run_after=2025-08-19 00:00:00+00:00
[2025-08-19T22:20:39.637+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.074 seconds
[2025-08-19T22:21:09.662+0000] {processor.py:161} INFO - Started process (PID=155) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:21:09.663+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:21:09.666+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:21:09.665+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:21:09.675+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:21:09.693+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:21:09.693+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:21:09.708+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:21:09.707+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-18 00:00:00+00:00, run_after=2025-08-19 00:00:00+00:00
[2025-08-19T22:21:09.718+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.059 seconds
[2025-08-19T22:21:40.505+0000] {processor.py:161} INFO - Started process (PID=211) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:21:40.508+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:21:40.510+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:21:40.510+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:21:40.521+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:21:40.544+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:21:40.544+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:21:40.559+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:21:40.559+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-18 00:00:00+00:00, run_after=2025-08-19 00:00:00+00:00
[2025-08-19T22:21:40.570+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.070 seconds
[2025-08-19T22:22:11.116+0000] {processor.py:161} INFO - Started process (PID=267) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:22:11.118+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:22:11.121+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:22:11.120+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:22:11.138+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:22:11.163+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:22:11.162+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:22:11.180+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:22:11.179+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-18 00:00:00+00:00, run_after=2025-08-19 00:00:00+00:00
[2025-08-19T22:22:11.190+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.085 seconds
[2025-08-19T22:22:41.633+0000] {processor.py:161} INFO - Started process (PID=323) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:22:41.638+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:22:41.641+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:22:41.641+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:22:41.653+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:22:41.675+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:22:41.675+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:22:41.690+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:22:41.690+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-18 00:00:00+00:00, run_after=2025-08-19 00:00:00+00:00
[2025-08-19T22:22:41.701+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.091 seconds
[2025-08-19T22:23:12.377+0000] {processor.py:161} INFO - Started process (PID=383) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:23:12.383+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:23:12.392+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:23:12.392+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:23:12.405+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:23:12.424+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:23:12.423+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:23:12.438+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:23:12.438+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:23:12.448+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.076 seconds
[2025-08-19T22:23:42.944+0000] {processor.py:161} INFO - Started process (PID=445) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:23:42.946+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:23:42.951+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:23:42.950+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:23:42.968+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:23:42.989+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:23:42.988+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:23:43.003+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:23:43.003+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:23:43.012+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.076 seconds
[2025-08-19T22:24:13.383+0000] {processor.py:161} INFO - Started process (PID=501) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:24:13.386+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:24:13.389+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:24:13.389+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:24:13.409+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:24:13.432+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:24:13.432+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:24:13.446+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:24:13.446+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:24:13.458+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.081 seconds
[2025-08-19T22:24:43.835+0000] {processor.py:161} INFO - Started process (PID=557) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:24:43.837+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:24:43.841+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:24:43.841+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:24:43.859+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:24:43.873+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:24:43.873+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:24:43.889+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:24:43.889+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:24:43.897+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.070 seconds
[2025-08-19T22:25:14.294+0000] {processor.py:161} INFO - Started process (PID=613) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:25:14.297+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:25:14.299+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:25:14.299+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:25:14.321+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:25:14.335+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:25:14.335+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:25:14.350+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:25:14.350+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:25:14.360+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.073 seconds
[2025-08-19T22:25:44.706+0000] {processor.py:161} INFO - Started process (PID=669) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:25:44.709+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:25:44.711+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:25:44.711+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:25:44.723+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:25:44.743+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:25:44.743+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:25:44.765+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:25:44.765+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:25:44.775+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.079 seconds
[2025-08-19T22:26:15.161+0000] {processor.py:161} INFO - Started process (PID=725) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:26:15.164+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:26:15.166+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:26:15.166+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:26:15.182+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:26:15.206+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:26:15.206+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:26:15.223+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:26:15.223+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:26:15.231+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.075 seconds
[2025-08-19T22:26:45.541+0000] {processor.py:161} INFO - Started process (PID=781) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:26:45.544+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:26:45.546+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:26:45.545+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:26:45.557+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:26:45.571+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:26:45.570+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:26:45.582+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:26:45.582+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:26:45.595+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.059 seconds
[2025-08-19T22:27:15.994+0000] {processor.py:161} INFO - Started process (PID=837) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:27:15.996+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:27:15.999+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:27:15.998+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:27:16.015+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:27:16.027+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:27:16.027+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:27:16.040+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:27:16.040+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:27:16.052+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.062 seconds
[2025-08-19T22:27:46.407+0000] {processor.py:161} INFO - Started process (PID=897) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:27:46.411+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:27:46.415+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:27:46.414+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:27:46.435+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:27:46.451+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:27:46.451+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:27:46.466+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:27:46.465+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:27:46.480+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.082 seconds
[2025-08-19T22:28:17.157+0000] {processor.py:161} INFO - Started process (PID=953) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:28:17.159+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:28:17.164+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:28:17.163+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:28:17.180+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:28:17.206+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:28:17.205+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:28:17.223+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:28:17.223+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:28:17.236+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.095 seconds
[2025-08-19T22:28:47.665+0000] {processor.py:161} INFO - Started process (PID=1009) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:28:47.666+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:28:47.669+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:28:47.668+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:28:47.683+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:28:47.698+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:28:47.698+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:28:47.709+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:28:47.709+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:28:47.720+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.058 seconds
[2025-08-19T22:29:18.116+0000] {processor.py:161} INFO - Started process (PID=1065) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:29:18.117+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:29:18.119+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:29:18.119+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:29:18.132+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:29:18.147+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:29:18.147+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:29:18.161+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:29:18.161+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:29:18.170+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.059 seconds
[2025-08-19T22:29:48.804+0000] {processor.py:161} INFO - Started process (PID=1121) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:29:48.807+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:29:48.809+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:29:48.809+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:29:48.823+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:29:48.842+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:29:48.842+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:29:48.855+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:29:48.855+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:29:48.865+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.066 seconds
[2025-08-19T22:30:19.259+0000] {processor.py:161} INFO - Started process (PID=1177) to work on /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:30:19.260+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/nyc_pipeline_dag.py for tasks to queue
[2025-08-19T22:30:19.264+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:30:19.264+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:30:19.275+0000] {processor.py:840} INFO - DAG(s) 'nyc_data_pipeline' retrieved from /opt/airflow/dags/nyc_pipeline_dag.py
[2025-08-19T22:30:19.297+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:30:19.297+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-08-19T22:30:19.320+0000] {logging_mixin.py:188} INFO - [2025-08-19T22:30:19.320+0000] {dag.py:3823} INFO - Setting next_dagrun for nyc_data_pipeline to 2025-08-19 00:00:00+00:00, run_after=2025-08-20 00:00:00+00:00
[2025-08-19T22:30:19.333+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/nyc_pipeline_dag.py took 0.079 seconds
